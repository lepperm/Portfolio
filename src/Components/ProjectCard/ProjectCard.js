/**
 * @fileoverview ProjectCard component.
 */

import React from "react";
import PropTypes from "prop-types";

/**
 * ProjectCard component
 *
 * @return {Component} ProjectCard component
 */
export const ProjectCard = () => {
  return <div>ProjectCard</div>;
};

/**
 * Prop types for ProjectCard component
 */
ProjectCard.propTypes = {
  //
};
