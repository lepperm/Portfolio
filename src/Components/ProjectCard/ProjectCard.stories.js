/**
 * @fileoverview Story for ProjectCard component.
 */

import React from "react";
import { ProjectCard } from "./ProjectCard";

export default {
  title: "ProjectCard",
  component: ProjectCard,
};

export const Basic = () => <ProjectCard />;
