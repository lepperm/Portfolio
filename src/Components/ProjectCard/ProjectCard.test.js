/**
 * @fileoverview Tests for ProjectCard component.
 */

import React from "react";
import { shallow } from "enzyme";
import { ProjectCard } from "./ProjectCard";

describe("ProjectCard", () => {
  test("matches snapshot", () => {
    const wrapper = shallow(<ProjectCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
