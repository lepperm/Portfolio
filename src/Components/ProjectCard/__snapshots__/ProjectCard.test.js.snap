// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectCard matches snapshot 1`] = `ShallowWrapper {}`;
